pkgs/development/python-modules/hikari/README.md

```diff
@@ -0,0 +1,5 @@
+# python3Packages.hikari
+## Updating
+The package build process requires a Git commit hash. This has been semi-automated by a script.
+
+Update the revision like normal, then run `./update_git_hash.sh $GIT_TAG`, where `$GIT_TAG` is the version number. This uses `sed` to update the Git hash that is specified at the top of `./default.nix`.
```

Comment:
Even if `git_hash` is not updated, the build itself will succeed.
This would be a problem with automatic updates using tools like `nixpkgs-update`.
I think getting the version in `update.sh` and set `passthru.update-script` would be better.

Reply:
Yes, it will succeed, but the build will not be correct. The documentation never said it would fail, only that building the package requires the SHA to be set.

Reply:
I mean, we know that there is a manual operation so that we can update properly.
However, in the typical case where "r-ryantm" updates and someone unaware of this operation approves and merges, the incorrect package is released because the build appears successful even though the "git_hash" is mismatched.
If the update-script is appropriately defined, we can avoid these errors.

Reply:
I *think* I did it right. My main concern, however, is that the version number will not be properly bumped.

Reply:
It would be great to update the version in `update.sh`.
I think that this updateScript does not work as expected.

If it is difficult, we can revert this change and keep it semi-automatic.

Reply:
> I think that this updateScript does not work as expected.

My other main hesitation was that I don't really know how `updateScript` works and what it does. [The manual section](https://nixos.org/manual/nixpkgs/unstable/#var-passthru-updateScript) feels a little light on the _how_.

Reply:
Yeah, bit that's due the fact, as there are several updaters around depending on what to do. I guess here a c&p from another update.sh from nixpkgs that loads a release from GH and your script after that should (not tested) give the expected result. You can run the updater also by your own, thus you can test if a update works properly, drop an older version im the default.nix, run the updater for the package and then check if it's the result you are expecting 
Running the updater should be something like
`nix-shell maintainers/scripts/update.nix --argstr package <exposed package name>` (for the package name I am not 100% sue how that is crafted for python packages in that case). I hope that helps a bit. 🙂

Reply:
I modified the existing update script to also update the version. Thank you for your help @natsukiem @Shawn8901!